### PR TITLE
Added mobile & accessible changes

### PR DESCRIPTION
**Pull request by indefinities**
Fixes hacks/hacks.github.io#65

### CODE REVIEW

File: src/components/hacks-nav.vue [hunk 1]
```diff
@@ -72,17 +72,16 @@ export default {
...
     display: block;
-    width: 100vw;
     top: 0;
     right: 0;
 
     h2 {
       margin: unset;
       text-align: right;
-      font-size: 108px !important;
+      font-size: $hacks-hamburger-size !important;
```
Comment (jennydaman) [inline src/components/hacks-nav.vue:84]: `!important` :cry:

File: src/styles/_variables.scss [hunk 2]
```diff
@@ -1,14 +1,36 @@
 // Text
-$hacks-h1: 48px;
-$hacks-h2: 36px;
-$hacks-text: 24px;
+$hacks-h1: 3.6em;
```
Comment (jennydaman) [inline src/styles/_variables.scss:2]: :+1:

File: src/styles/_variables.scss [hunk 3]
```diff
@@ -1,14 +1,36 @@
...
+      $alpha: 0.4,
+    ),
+  -2px 0px rgba(
+      $color: cyan,
+      $alpha: 0.4,
+    );
 
 // Sizes
-$hacks-page-width: 981px;
+$hacks-page-width: em(981);
```
Comment (jennydaman) [inline src/styles/_variables.scss:33]: fancy sass :open_mouth:

Comment (indefinities) [reply src/styles/_variables.scss:33]: undid it because it don't work doe

Comment (jennydaman) [reply src/styles/_variables.scss:33]: you baited me into reading https://adamwathan.me/dont-use-em-for-media-queries/

Comment (jennydaman) [reply src/styles/_variables.scss:33]: wait where's the undo

Comment (indefinities) [reply src/styles/_variables.scss:33]: I'm going to scream brb

**Comment by indefinities** [comment]
fixed desktop version -- it was because of container widths

**Comment by jennydaman** [comment]
maybe you're going tor the picture book aesthetic

**Comment by indefinities** [comment]
I don't know what aesthetic I'm going for please

**Pull request merged**